### PR TITLE
[gstreamer] Fix building on linux

### PR DESCRIPTION
**Pull request by m-kuhn**
**Describe the pull request**

- #### What does your PR fix?
  Fixes building gstreamer on linux

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  All

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yeah

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  trust the bots

### CODE REVIEW

**Comment by m-kuhn** [comment]
That was an interesting no-op force push to get the github-actions check happy :dancers:

File: ports/gstreamer/portfile.cmake [hunk 1]
```diff
@@ -325,6 +327,19 @@ if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
     file(REMOVE ${DBG_BINS} ${REL_BINS})
 endif()
```
Comment (Neumann-A) [inline ports/gstreamer/portfile.cmake:329]: Check the qt3d pr. This is also required for static windows builds + a change in x264 linkage behavior. Also the qt3d pr has a simpler approach for moving the files.

Comment (m-kuhn) [reply ports/gstreamer/portfile.cmake:329]: Looks like I can close this as duplicate of https://github.com/microsoft/vcpkg/pull/26928/files

Comment (Neumann-A) [reply ports/gstreamer/portfile.cmake:329]: #26928 will probably take some time. Feel free to copy/extract the changes.

Comment (m-kuhn) [reply ports/gstreamer/portfile.cmake]: Done

**Comment by m-kuhn** [comment]
Closing in favor of https://github.com/microsoft/vcpkg/pull/26928

**Comment by dg0yt** [comment]
Actually fixed in #27050 which was merged yesterday.

**Pull request closed without merging**